apps/httpd/main.cc

```diff
@@ -73,8 +73,6 @@ void set_routes(routes& r) {
 }
 
```

Comment:
> httpd: rm unused prom structs

prometheus

Reply:
also please consider using "remove" instead of "rm" for better readability. For example:

> httpd: remove unused prometheus structs

while brevity is appreciated, full words often enhance clarity in commit messages without significantly impacting title length.